hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/ContractsConfig.java

```diff
@@ -40,6 +40,8 @@ public record ContractsConfig(
         // "TokenAssociateToAccount,TokenDissociateFromAccount,TokenFreezeAccount,TokenUnfreezeAccount,TokenGrantKycToAccount,TokenRevokeKycFromAccount,TokenAccountWipe,TokenBurn,TokenDelete,TokenMint,TokenUnpause,TokenPause,TokenCreate,TokenUpdate,ContractCall,CryptoTransfer") Set<HederaFunctionality> allowSystemUseOfHapiSigs,
         @ConfigProperty(defaultValue = "0") @NetworkProperty long maxNumWithHapiSigsAccess,
         // @ConfigProperty(defaultValue = "") Set<Address> withSpecialHapiSigsAccess,
+        @ConfigProperty(value = "nonces.externalization.enabled", defaultValue = "true") @NetworkProperty
```

Comment:
(Impossible to tell if my previous comment got disappeared when this file got updated:)

(I need to understand the new config system better, because: Why do many of these properties not have a `value` (i.e., external name)?)

Reply:
I think we only need to specify `value` if the property's name can't be constructed by dot-joining the config class's `@ConfigData` annotation value with the Java field name of the property.

So for example, given:
```
@ConfigData("contracts")
...
@ConfigProperty(defaultValue = "0") @NetworkProperty long maxNumWithHapiSigsAccess,
```
this automatically implies a property of,
```
contracts.maxNumWithHapiSigsAccess=0
```

But this nonce externalization property doesn't follow this pattern.